code/game/machinery/scan_gate.dm

```diff
@@ -50,12 +55,18 @@
 	else
 		. += "<span class='notice'>The control panel is unlocked. Swipe an ID to lock it.</span>"
 
+/obj/machinery/scanner_gate/update_icon_state()
+	if(panel_open)

```

Comment:
What is the point of this

Reply:
I can remove this or we can leave it if someone plans to implement a maintenance mode sprite